src/main/native/CCM.c

```diff
@@ -385,17 +385,17 @@ int CCM_encrypt_core(JNIEnv* env, ICC_CTX* ockCtx,
     }
 #ifdef DEBUG_CCM_DATA
     if ( debug ) {
-        gslogMessagePrefix ("DATA_CCM ivNative : ");
-        gslogMessageHex (ivNative, 0, (int) ivLen, 0, 0, NULL);
+        gslogMessagePrefix ("DATA_CCM iv : ");
+        gslogMessageHex ((char*)iv, 0, (int) ivLen, 0, 0, NULL);
 
-        gslogMessagePrefix ("DATA_CCM keyNative : ");
-        gslogMessageHex (keyNative, 0, (int) keyLen, 0, 0, NULL);
+        gslogMessagePrefix ("DATA_CCM key : ");
+        gslogMessageHex ((char*)key, 0, (int) keyLen, 0, 0, NULL);
 
-        gslogMessagePrefix ("DATA_CCM iphertextNative : ");
-        gslogMessageHex (plaintextNative, 0, (int) plaintextLen, 0, 0, NULL);
+        gslogMessagePrefix ("DATA_CCM plain : ");
```

Comment:
Maybe `plainText` would be a better name?

Reply:
In the various debugs I would prefer to use the name of the variables instead of using `plainText` and plain wording for the debugs as it makes it easier to search through the code later looking for what was printed in a given debug log. In this case the variable `plainText` does not exist in this method.

Reply:
If you ask me, the variable should also be named `plainText`, as `plain` is not really accurate. Would it make sense to change it, or should we just leave it for another PR?

Reply:
OK yes agreed that the variable names `plainText` and `cipherText` are better names then `plain` and `cipher` that is in use in this method now. I updated the names using commit https://github.com/IBM/OpenJCEPlus/pull/318/commits/b0541ed0c25c119cf7c159dfe68393f745687886